### PR TITLE
Make the code clear and easy to add functions

### PR DESCRIPTION
**Pull request by misssalad612**
Hello! This is a cool program that help me a lot, so I want to do something to help improving it.

 The followings below are some changes I made:
- Move the conversion code to ConvertSource.java
- Divide the full conversion process by tags
- Add ability to convert Height Map and Biomes

btw I found a site which shows almost all meaning of each tag in MCPE LevelDb.
It may be useful for programming, i think :)
https://minecraft.gamepedia.com/User:Jocopa3/Pocket_Level_Format_Test#LevelDB_Format

### CODE REVIEW

**Comment by thebigsmileXD** [comment]
I want to remind, if it matters: Light is not saved into the level data anymore, but calculated within 0.8 seconds while loading chunks according to tommaso.

File: src/com/ljyloo/PE2PC/UnsharedData.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+			return new byte[]{(byte) 3, 2};
+		case (byte) 244:  //beetroot
+			return new byte[]{(byte) 207, data};
+		case (byte) 245:  //stonecutter
+		case (byte) 246:  //glowingobsidian
+		case (byte) 247:  //netherreactor
+		case (byte) 248:  //info_update
+		case (byte) 249:  //info_update2
+		case (byte) 250:  //movingblock
+		case (byte) 251:  //observer
```
Comment (thebigsmileXD) [inline src/com/ljyloo/PE2PC/UnsharedData.java]: Actually, these exist in MCJava

**Comment by misssalad612** [comment]
That's OK. That's why I chose to ignore light data whether it exists.

File: src/com/ljyloo/PE2PC/UnsharedData.java (hunk 1 above)
Comment (misssalad612) [inline src/com/ljyloo/PE2PC/UnsharedData.java]: I forgot to check it, already fixed :)

**Comment by ljyloo** [comment]
Huge thanks to you guys! Really appreciate you guys do the refactoring job for me! And nice code compare with my messy code:) I am not a great programmer and I program just for fun, thanks for helping me out

**Pull request merged**